documentation/docs/faq/liquidity-mining.md

```diff
@@ -100,59 +100,96 @@ The Hummingbot Miners app uses your Ethereum wallet address to:
-3. Click the plus sign next to My **Wallets** to add a Zcoin wallet. Type **Zcoin** or select it from the list.  
-4. Choose your desired fiat pair, then name you wallet and confirm by clicking "Create Wallet".  
-5. Zcoin wallet will appear on the main screen, click Zcoin wallet then click **Request**. This generates an address with a correspoind QR code which you can copy and share to receive your coin.  
+!!! note
+    - You may also refer to this step on [How to get your deposit address on Binance](https://academy.binance.com/tutorials/how-to-deposit#deposit). 
+    - You can also check your ZCoin wallet on https://explorer.zcoin.io/
+    - `Zcoin address start with:  "a..." or "Z..." or "3..." for multi-sig wallets.`
+
+### Get Zcoin deposit address with mobile wallet
+
+1. Download edge Wallet from Apple Store or Google Play.
+2. Open the app and create an account - complete the sign up process.
+3. Click the plus sign next to My **Wallets** to add a Zcoin wallet. Type **Zcoin** or select it from the list.
+4. Choose your desired fiat pair, then name you wallet and confirm by clicking "Create Wallet".
+5. Zcoin wallet address will appear on the main screen, click Zcoin wallet then click **Request**. This generates an address with a correspoind QR code which you can copy and share to receive your coin.
 
 Above steps are for new user, existing user can jump to step #5 after successful login.
 
-!!! note "Note"
+!!! note
     Though above example uses edge wallet, Zcoin doesn't endorse one wallet in particular.
-    You may also refer to this step on [How to receive Zcoin with a mobile wallet](https://zcoin.io/how-to-receive-zcoin-with-a-mobile-wallet) 
-    And also this step on [Installation and and getting started with your Zcoin wallet](https://youtu.be/8-HKyM3QrJU)   
-      
+
+    - You may also refer to this step on [How to receive Zcoin with a mobile wallet](https://zcoin.io/how-to-receive-zcoin-with-a-mobile-wallet)
+    - And also this step on [Installation and and getting started with your Zcoin wallet](https://youtu.be/8-HKyM3QrJU)
+
+### Get NEM deposit address
+
+1. Go to [NEM Wallet](https://nemplatform.com/wallets/).
+2. Choose the option that is supported by your OS.
+3. Launch the wallet.
+4. Create an account - complete the sign up process.
+5. Login to your newly created wallet.
+6. NEM wallet address will appear on the main screen, under Account Information.
+
+You may also refer to this step on [how to create NEM wallet](https://youtu.be/itEcOd6ABrA).
+
+### Get Algorand deposit address from Algorand Web Wallet
+
+1. Go to [Algorand Web Wallet](https://myalgo.com/).
+2. Create an account - complete the sign up process.
+3. Make sure to save the mnemonic phrase. You need this to recover your wallet.
+4. Your wallet address will appear on the main screen, below your wallet name.
+
+### Adding Tether USDT to your Algorand wallet in Algorand Web Wallet
```

Comment:
Enabling Tether USDT to your Algorand wallet in Algorand Web Wallet

Important note: you must enable USDT on the Algo wallet in order to be able to receive USDT ASA.  Enabling USDT will also require you to have some amount of ALGO tokens in the wallet in order to call the function to enable USDT.